### ISSUE TITLE
Can this sniff bluetooth classic pairing?

### ISSUE THREAD

**Issue by PowerPress** [Matheus-Garbelini/esp32_bluetooth_classic_sniffer#7]
Can this sniff bluetooth classic pairing?

**Comment by Matheus-Garbelini** [comment]
Hi @PowerPress it certainly can. You can check the example wireshark capture on [logs/sample](https://github.com/Matheus-Garbelini/esp32_bluetooth_classic_sniffer/tree/master/logs/samples) folder.

**Comment by PowerPress** [comment]
Great does this automatically do frequency hopping? How many esp's do you need to do this? Goal is to be able demonstrate threat of recording pairing without using 10-20k full spectrum analyzers.

**Comment by Matheus-Garbelini** [comment]
@PowerPress yes, this does frequency hopping. However, this sniffer takes part in the communication,
so it cannot sniff passevelly the communication between two BT devices but rather, it communicates with the other devices while  dumping it's baseband communication to the PC.

**Comment by PowerPress** [comment]
Curious what if you had 1 for every channel all recording simultaneously to using this and saving to wireshark. Would this project support that if I had the esp32's? I just want to record full spectrum on the chance of catching a bluetooth pairing in pcap type format.

Also do you know how I could do the samething for ble le?

**Comment by pkarashchenko** [comment]
I need something similar. I have two devices communication between each other and need a way to sniff that communication (passive sniffing), @Matheus-Garbelini is there a plan to add passive sniffer mode?
